packages/verification/README.md

```diff
@@ -87,19 +90,22 @@ const credentials: ApplicationCredentials = {
 const verificationService = new VerificationService(credentials);
 
 // Build the request data
-const requestData: Verification.StartVerificationRequestData = {
-  initiateVerificationRequestBody: {
+const requestData: Verification.StartSmsVerificationRequestData = {
+  startVerificationWithSmsRequestBody: {
     identity: {
       type: 'number',
       endpoint: '+17813334444',
     },
-    method: 'sms',
+    smsOptions: {
+      codeType: 'Alphanumeric',
+      locale: 'sv-SE',
+    },
   },
 };
 
 // Use the standalone declaration of the 'verification' service
-const verificationInitResponse: Verification.InitiateVerificationResponse 
-    = await verificationService.verifications.start(requestData);
+const startVerificationResponse: Verification.StartSmsVerificationResponse 
+    = await verificationService.startVerifications.startSms(requestData);
```

Comment:
Not fan of renaming `verificationService.verifications`  to `verificationService.startVerifications`
It could have sense ( with a major breaking change), but it start to drift with other SDK's provided implementation and ID <> SDKs function mapping

Reply:
Indeed, I generated from the OAS tag, I'll update according to the confluence page